fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1186,6 +1186,9 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 	                                                          span.context,
 	                                                          self->debugID);
 
+	float ratio = self->toCommit.getEmptyLocationRatio();
```

Comment:
I think `getEmpyMessageRatio` is more more descriptive / matches stat name

Reply:
Done.